### PR TITLE
SMILE-699-preview-iframes-content-url-as-src, functionality implemented

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

File: src/components/ContentPreview.tsx [hunk 1]
```diff
@@ -85,6 +90,19 @@ const ContentPreview: FC<ContentPreviewProps> = ({ resource }) => {
           scrolling="no"
         />
       )}
+      {(presentationMode === SupportedFileTypes.Youtube ||
```
Comment (anetteOlli) [inline src/components/ContentPreview.tsx:93]: Kjørte mot prod for å sjekke at det fungerte. 
Youtube: http://localhost:3000/resource/d39d7ae7-308a-49d2-8052-9416a982bd5e
kaltura: http://localhost:3000/resource/2b803717-e280-4825-bdf2-fbd22d33878d
vimeo: http://localhost:3000/resource/9e576880-af90-47ee-b4d1-bd95e4e3711a
MediaSite: http://localhost:3000/resource/1cc3312a-30eb-470a-bf9b-f5bfd2fb0000
link: http://localhost:3000/resource/838437fe-e42e-4f80-9bad-5ba4a057bc7b

Comment (anetteOlli) [reply src/components/ContentPreview.tsx:95]: Spotify ressurs på dev: http://localhost:3000/resource/ea405ccf-ea42-496b-8990-5ddb120e73c7

File: src/components/ContentPreview.tsx [hunk 2]
```diff
@@ -46,7 +46,14 @@ const ContentPreview: FC<ContentPreviewProps> = ({ resource }) => {
...
         !(presentationMode === SupportedFileTypes.Audio) &&
-        !(presentationMode === SupportedFileTypes.PDF) && (
+        !(presentationMode === SupportedFileTypes.PDF) &&
+        !(presentationMode === SupportedFileTypes.Kaltura) &&
+        !(presentationMode === SupportedFileTypes.Youtube) &&
+        !(presentationMode === SupportedFileTypes.MediaSite) &&
+        !(presentationMode === SupportedFileTypes.Link) &&
+        !(presentationMode === SupportedFileTypes.Vimeo) &&
+        !(presentationMode === SupportedFileTypes.Download) &&
+        !(presentationMode === SupportedFileTypes.Spotify) && (
```
Comment (pcb-unit) [inline src/components/ContentPreview.tsx:56]: samle disse og de under i en funksjon isPreviewSupported(presentationMode) elns ?

Comment (anetteOlli) [reply src/components/ContentPreview.tsx]: Når 23video og soundCloud er implementert, så skal hele denne blokka bort.

**Review by pcb-unit — APPROVED** [review]

**Pull request merged**